google/cloud/storage/client.h

```diff
@@ -2252,6 +2253,81 @@ class Client {
     return GetServiceAccountForProject(project_id,
                                        std::forward<Options>(options)...);
   }
+
+  /**
+   * Create a new HMAC key in a given project.
+   *
+   * @warning This GCS feature is not GA, it is subject to change without
+   *     notice.
+   *
+   * @param project_id the name of the project where you want to create the
+   *     service account.
+   * @param service_account the service account email where you want to create
+   *     the new HMAC key.
+   * @param options a list of optional query parameters and/or request headers.
+   *     Only the standard options apply in this case.
+   *
+   * @return This operation returns the new HMAC key metadata *and* the HMAC key
+   *   secret (encoded as a base64 string). This is the only request that
+   *   returns the secret.
+   *
+   * @par Idempotency
+   * This operation is not idempotent. Retrying the operation will create a new
+   * key each time.
+   *
+   * @par Example
+   *
+   *
+   * @see https://cloud.google.com/iam/docs/service-accounts for general
+   *     information on Google Cloud Platform service accounts.
+   */
+  template <typename... Options>
+  StatusOr<std::pair<HmacKeyMetadata, std::string>> CreateHmacKeyForProject(
+      std::string project_id, std::string service_account,
+      Options&&... options) {
+    internal::CreateHmacKeyRequest request(std::move(project_id),
+                                           std::move(service_account));
+    request.set_multiple_options(std::forward<Options>(options)...);
+    auto result = raw_client_->CreateHmacKey(request);
+    if (!result) {
+      return result.status();
+    }
+    return std::make_pair(std::move(result.value().resource),
```

Comment:
We should probably eschew `.value()` for `->` in non-example code (where we should have always checked the status, like here).

Reply:
Fixed.